### PR TITLE
 New governance section

### PR DESCRIPTION
**Pull request by ramGranell**
This is related to  #1967 You can check the results at the bottom of https://deploy-preview-1968--fairsharing.netlify.app/communities#governance

### CODE REVIEW

**Comment by allysonlister** [comment]
Thank you - looks great! I have now discovered the orcid for Graham - it's https://orcid.org/0000-0001-9520-0109

**Comment by ramGranell** [comment]
> Thank you - looks great! I have now discovered the orcid for Graham - it's https://orcid.org/0000-0001-9520-0109

Added this ORCID in the last version

**Review by allysonlister — APPROVED** [review]
thanks!

File: src/data/communityPageData.json [hunk 1]
```diff
@@ -343,5 +343,27 @@
         ]
       }
     ]
+  },
+  "rda": {
+    "title": "RDA FAIRsharing WG",
+    "description": "\n\n   The Research Data Alliance <a href='https://www.rd-alliance.org/group/fairsharing-registry-connecting-data-policies-standards-databases.html' target='_blank'>FAIRsharing Registry: Connecting data policies, standards and databases</a> Working Group (WG) was created to promote further adoption and drive improvement of the FAIRsharing registry. The WG contributes to a number of activities with other RDA groups; these activities reflect the requests and needs of the community, and are open and inclusive for all stakeholders. The WG community network encompasses repository mantainers and developers, standard-developing organizations, journals and journal publishers, funders, librarians, data stewards, and other scholarly organizations and alliances.\n",
```
Comment (prakhyatox) [inline src/data/communityPageData.json]: Please remove \n\n from the beginning of the sentence

File: src/data/communityPageData.json [hunk 2]
```diff
@@ -343,5 +343,27 @@
...
+          {"link": "https://orcid.org/0000-0003-2687-1982", "text": "<b>Peter McQuilton, </b>GSK"},
+          {"link": "https://orcid.org/0000-0002-7614-0806", "text": "<b>Rebecca Grant, </b>F1000"},
+          {"link": "https://orcid.org/0000-0003-3179-7270", "text": "<b>Simon Hodson, </b>CODATA"},
+          {"link": "https://orcid.org/0000-0002-7702-4495", "text": "<b>Allyson Lister, </b>University of Oxford"},
+          {"link": "https://orcid.org/0000-0001-5306-5690", "text": "<b>Susanna-Assunta Sansone, </b>University of Oxford"}
+        ]
+      },
+      {
+        "title": "Members",
+        "data": "View our members on the <a href='https://www.rd-alliance.org/node/44787/members' target='_blank'>RDA FAIRsharing WG members' pages</a>.\n"
```
Comment (prakhyatox) [inline src/data/communityPageData.json]: Please remove ```\n``` from the end

**Review by prakhyatox — APPROVED** [review]

**Pull request merged**